### PR TITLE
FETraining Profile Card

### PR DESCRIPTION
**Pull request by dkdeepesh**
Reference:-
image

Result:-
Desktop:-
image
Desktop(on hover):
image
image
image
Mobile:-
image

### CODE REVIEW

**Review by mathrik — COMMENTED** [review]
It's not clear from your specs, but I would expect to see some padding above the image on your card. It looks almost clipped the way it is now

File: Assets/scss/components/_profile-card.scss [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+
+            &__image {
+                width: 150px;
+                height: 150px;
+                margin: 0 auto;
+                border-radius: 50%;
+                overflow: hidden;
+
+                img {
+                    width: 100%;
```
[the remark this answers is not in the document]
Comment (dkdeepesh) [reply Assets/scss/components/_profile-card.scss:52]: Sure, I will implement the suggested changes

[the remark this answers is not in the document]
Comment (dkdeepesh) [reply Assets/scss/components/_profile-card.scss]: image
Did changes and added margin top to the image

File: Assets/scss/components/_profile-card.scss [hunk 2]
```diff
@@ -0,0 +1,97 @@
+.cmp-profile-card {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    min-height: 100vh;
```
[the remark this answers is not in the document]
Comment (dkdeepesh) [reply Assets/scss/components/_profile-card.scss]: I was just trying to see what happened when I applied this and forgot to remove it before committing the code.

**Review by mathrik — APPROVED** [review]
This looks fine to me. Just want to make sure @jwetmore-hoodoo has a look before we merge it.

File: Assets/scss/components/_profile-card.scss [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+    &__container {
+        display: flex;
+        flex-wrap: wrap;
+        gap: 20px;
+        justify-content: center;
+        padding: 20px;
+
+        &__profile {
+            width: 300px;
+            border: 1px solid #ccc;
```
Comment (jwetmore-hoodoo) [inline Assets/scss/components/_profile-card.scss:17]: Color could be moved to _vars file

File: Assets/scss/components/_profile-card.scss [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+
+                .light & {
+                    color: $sonic-silver;
+                }
+            }
+
+            &__title,
+            &__company {
+                margin: 5px 0;
+                font-size: 16px;
```
Comment (jwetmore-hoodoo) [inline Assets/scss/components/_profile-card.scss:73]: All font-sizes could use `toRem()` function

File: Assets/scss/components/_profile-card.scss (hunk 3 above)
Comment (dkdeepesh) [reply Assets/scss/components/_profile-card.scss]: I have done the changes.

File: Assets/scss/components/_profile-card.scss (hunk 4 above)
Comment (dkdeepesh) [reply Assets/scss/components/_profile-card.scss]: I have done the changes.

**Comment by mathrik** [comment]
Updates to using rems and variables look good.

**Comment by dkdeepesh** [comment]
I have created a new PR for the above solution, incorporating all the required changes as requested.
[FETraining Profile Card by dkdeepesh · Pull Request #33 · jvaishnavrp/FETraining (github.com)](https://github.com/jvaishnavrp/FETraining/pull/33)
cc: - @mathrik @jwetmore-hoodoo @Lalit-gour1 @jvaishnavrp

**Comment by mathrik** [comment]
I merged 33, but this PR has conflicts that need to be resolved (or just close this one)

**Pull request closed without merging**